<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Project_DM_Qt</class>
 <widget class="QMainWindow" name="Project_DM_Qt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1110</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1110</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Project_DM_Qt</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>300</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>240</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>240</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>300</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>290</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>*</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>/</string>
     </property>
    </item>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>420</y>
      <width>104</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="frame_Brn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>720</y>
      <width>1101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="Btn_natural">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: #000000;
background-color: white;
border: 1px solid #828282;
border-radius: 3px;
font-size: 18px;
}
QPushButton:hover {
border: 1px solid #828282;
background-color: #d5d5d5;
}
QPushButton:hover:pressed { 
background-color: gray;
color: white;
}
QPushButton:disabled { 
background-color: gray;
color: white;
}</string>
     </property>
     <property name="text">
      <string>Натуральные числа N</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_integer">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: #000000;
background-color: white;
border: 1px solid #828282;
border-radius: 3px;
font-size: 18px;
}
QPushButton:hover {
border: 1px solid #828282;
background-color: #d5d5d5;
}
QPushButton:hover:pressed { 
background-color: gray;
color: white;
}
QPushButton:disabled { 
background-color: gray;
color: white;
}</string>
     </property>
     <property name="text">
      <string>Целые числа Z</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_rational">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>10</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: #000000;
background-color: white;
border: 1px solid #828282;
border-radius: 3px;
font-size: 18px;
}
QPushButton:hover {
border: 1px solid #828282;
background-color: #d5d5d5;
}
QPushButton:hover:pressed { 
background-color: gray;
color: white;
}
QPushButton:disabled { 
background-color: gray;
color: white;
}</string>
     </property>
     <property name="text">
      <string>Рациональные числа Q</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_polynomials">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>10</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: #000000;
background-color: white;
border: 1px solid #828282;
border-radius: 3px;
font-size: 18px;
}
QPushButton:hover {
border: 1px solid #828282;
background-color: #d5d5d5;
}
QPushButton:hover:pressed { 
background-color: gray;
color: white;
}
QPushButton:disabled { 
background-color: gray;
color: white;
}</string>
     </property>
     <property name="text">
      <string>Многочлены P</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_matrix">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>10</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: #000000;
background-color: white;
border: 1px solid #828282;
border-radius: 3px;
font-size: 18px;
}
QPushButton:hover {
border: 1px solid #828282;
background-color: #d5d5d5;
}
QPushButton:hover:pressed { 
background-color: gray;
color: white;
}
QPushButton:disabled { 
background-color: gray;
color: white;
}</string>
     </property>
     <property name="text">
      <string>Матрицы M</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1110</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>Тест</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
